shimmingtoolbox/shim/sequencer.py

```diff
@@ -82,12 +82,12 @@ def __init__(self, slices, mask_dilation_kernel, mask_dilation_kernel_size, reg_
         self.path_output = path_output
         self.optimizer = None
 
-    def optimize(self, resampled_mask):
+    def optimize(self, masks_fmap_dilated):
         """
         Optimization of the currents for each shim group. Wraps :meth:`shimmingtoolbox.shim.sequencer.Sequencer.opt`.
 
         Args:
-            resampled_mask (np.ndarray): 3D anat mask used for the optimizer to shim in the region of interest.
+            masks_fmap_dilated (np.ndarray): 3D anat mask used for the optimizer to shim in the region of interest.
```

Comment:
Is this right, the name says that it is the fmap mask and the description says that it is the anat

Reply:
good catch, I will fixed this